### ISSUE TITLE
Minor note: could use \figalt to add an "alt"

### ISSUE THREAD

**Issue by tlienart** [cserteGT3/JuDocPlottest#1]
If you use `\fig{path}` it corresponds to `![](path)` (no alt) whereas if you use `\figalt{alt}{path}` it corresponds to `![alt](path)`. 
You may be well aware of this, I just saw that in your example you used `\fig{...}` while elsewhere you use `![the plot](...)`, if you wanted identical effect it should be `\figalt{the plot}{...}` 🤷‍♂ 

(a bit of hair-splitting, sorry, feel free to close this, and thanks again for the comparison)

**Comment by cserteGT3** [comment]
Thank you for pointing out! I didn't understand the difference (now I googled it). I've updated the code.
But I encountered another issue, namely the text does not starts on a new line (while does it on the Literate and Weave pages):
![judoc](https://user-images.githubusercontent.com/26418354/60171137-4bc7f000-980a-11e9-8f9d-88642f3d578a.PNG)

I thought maybe putting a newline in the `println()` (in `src/pages/menu2.md`) would solve this, but no
```
...
println('\n', "Mean temperature: ", mean(d[:,2]), " °C")
```
it gaves the same result. Do you have any idea why this happens here and not on the Weave and Literate pages?

**Comment by tlienart** [comment]
Yeah so that one is pretty easy to explain and it could be fixed easily too. Basically at the moment the input is 

```
(preceding html)
inserted html
(following html)
```

whereas I suspect that weave, literate and what you expect would be:

```
(preceding html)
<p>inserted html</p>
(following html)
```

The whole difference is the `<p>...</p>`. Now the reason I did not directly add them when adding `\textoutput` is because I wasn't sure that we wanted a new paragraph everytime `\textoutput` was used. I.e. if I add them, then you will get the expected behaviour but it will not be possible to do something like:

```
Some markdown with the temperature:  \textoutput{...} and the end of the sentence.
```

because that will appear as

```
Some markdown with the temperature:

15C

and the end of the sentence
```

I hope this makes sense. The way forward would be either to:

* add the `<p>...</p>` anyway and assume that the case mentioned above will not be used as such, or
* leave as is (so that it can be used inline) and introduce yet another command `\textoutputp` with `\newcommand{\textoutputp}[1]{~~~<p>~~~\textoutput{#1}~~~</p>~~~}` which does what you were expecting

If you have an opinion I'll gladly take it. I guess here it starts showing that many convenience commands can be defined but that eventually naming issues will become cumbersome. This is why I would lean towards providing things that don't do too force too much formatting and add simple ways to extend the formatting. However others may expect behaviour like you did in which case there's a tradeoff between flexibility and intuitive usage. What do you think?

Sorry for the long answer and hope it makes sense!

PS: this also makes me think that it could be nice to add a command `\\` like in latex which would just force the inclusion of a `<p></p>`.

**Comment by cserteGT3** [comment]
Oh I see. I'm not an html or SSG expert in any means, so thanks for the explanation!
I agree with you regarding the `\textoutput` and I think that it should stay as-is.

**Comment by tlienart** [comment]
FYI, you can now use `\\` to force a line skip somewhere which should hopefully help with such situations :)
